engine/runtime/src/main/java/org/enso/interpreter/runtime/warning/GetAllWarningsNode.java

```diff
@@ -0,0 +1,73 @@
+package org.enso.interpreter.runtime.warning;
+
+import com.oracle.truffle.api.CompilerDirectives.TruffleBoundary;
+import com.oracle.truffle.api.dsl.Cached;
+import com.oracle.truffle.api.dsl.Cached.Shared;
+import com.oracle.truffle.api.dsl.Fallback;
+import com.oracle.truffle.api.dsl.Specialization;
+import com.oracle.truffle.api.interop.InteropLibrary;
+import com.oracle.truffle.api.interop.UnsupportedMessageException;
+import com.oracle.truffle.api.library.CachedLibrary;
+import com.oracle.truffle.api.nodes.Node;
+import java.util.Arrays;
+import java.util.Comparator;
+import org.enso.interpreter.dsl.AcceptsWarning;
+import org.enso.interpreter.dsl.BuiltinMethod;
+import org.enso.interpreter.runtime.EnsoContext;
+import org.enso.interpreter.runtime.data.hash.HashMapInsertNode;
+import org.enso.interpreter.runtime.data.vector.ArrayLikeAtNode;
+import org.enso.interpreter.runtime.data.vector.ArrayLikeHelpers;
+import org.enso.interpreter.runtime.data.vector.ArrayLikeLengthNode;
+
+@BuiltinMethod(
+    type = "Warning",
+    name = "get_all_vector",
+    description = "Gets all the warnings associated with the value.",
+    autoRegister = false)
+public abstract class GetAllWarningsNode extends Node {
+
+  public static GetAllWarningsNode build() {
+    return GetAllWarningsNodeGen.create();
+  }
+
+  public abstract Object execute(@AcceptsWarning Object value, boolean shouldWrap);
```

Comment:
`executeAllWarnings` maybe?

Reply:
`BuiltinMethodProcessor` cannot handle methods with different name: it expects there is an `execute` method:
```
[error] /home/pavel/dev/enso/engine/runtime/src/main/java/org/enso/interpreter/runtime/warning/GetAllWarningsNode.java:27:1:  error: No execute method found.
[error] public abstract class GetAllWarningsNode extends Node {
[error]                 ^1 error
[warn] frgaal exited with exit code 1
```